### PR TITLE
Payout docs fixes

### PR DESCRIPTION
**Pull request by koirikivi**
Some fixes/stuff for future.

This is untested right now so please test before merging or wait.

### CODE REVIEW

File: sto/ethereum/utils.py [hunk 1]
```diff
@@ -347,18 +350,21 @@ def get_contract_deployed_tx(dbsession, contract_name):
     ).first()
 
 
-def whitelist_kyc_address(config, address, kyc_contract_address):
+def whitelist_kyc_address(config, address, kyc_contract_address,
+                          do_broadcast=True,
```
Comment (voith) [inline sto/ethereum/utils.py]: Although `do_broadcast` is set here to true to reflect previous functionality, lets it set to false by default.
The rationale behind this is that we want to change the pattern used in this code. It shouldn't broadcast by itself.

File: tests/cli/test_cli.py [hunk 2]
```diff
@@ -280,7 +280,9 @@ def test_kyc_manage(
         ]
     )
     assert result.exit_code == 0
-    assert kyc_contract.functions.isWhitelisted(eth_address).call() == True
+    # TODO: this method is no longer supported but maybe this should be tested
```
Comment (voith) [inline tests/cli/test_cli.py]: This has changed in the new abi. You should check the `getAttribute` method in the new ABI. cc @villesundell .

File: sto/ethereum/issuance.py [hunk 3]
```diff
@@ -116,7 +116,8 @@ def contract_status(logger: Logger,
         logger.info("Decimals: %d", contract.functions.decimals().call())
         logger.info("Owner: %s", contract.functions.owner().call())
         logger.info("Broadcast account token balance: %f", normal_balance)
-        logger.info("Transfer verified: %s", contract.functions.transferVerifier().call())
+        # TODO: this seems to be no longer supported?
+        # logger.info("Transfer verified: %s", contract.functions.transferVerifier().call())
```
Comment (voith) [inline sto/ethereum/issuance.py]: @villesundell Any idea what changed here?

File: sto/ethereum/utils.py (hunk 1 above)
Comment (koirikivi) [reply sto/ethereum/utils.py]: Changed. Ty!

**Pull request merged**